apps/desktop/desktop/src/main/java/bisq/desktop/components/controls/MaterialTextField.java

```diff
@@ -186,18 +186,22 @@ public void setValidators(ValidatorBase... validators) {
         validationControl.setValidators(validators);
     }
 
-    // TODO add custom errorLabel and not reuse helpLabel as it would cause an exception when binding at the helpLabel is used
+
     public boolean validate() {
         isValid.set(validationControl.validate());
         selectionLine.pseudoClassStateChanged(PSEUDO_CLASS_ERROR, !isValid.get());
         descriptionLabel.pseudoClassStateChanged(PSEUDO_CLASS_ERROR, !isValid.get());
         getActiveValidator().ifPresentOrElse(validator -> {
                     errorLabel.setText(validator.getMessage());
                     errorLabel.pseudoClassStateChanged(PSEUDO_CLASS_ERROR, true);
+                    helpLabel.setOpacity(0);
```

Comment:
Better use setManaged(false) and setVisible(false) as that takes the component out of the render cycle and is better for performance. We use opacity only for dimming/fade effect and in some rare cases where setManaged causes too much troubles.

Reply:
Yes suggestion makes sense for optimization. Note that If i use setManaged(false) and setVisible(false) here, when you click out with mouse update is triggered which sets the help text to visible again making error message and helptext overlap

Update():

    helpLabel.setVisible(StringUtils.isNotEmpty(helpProperty().get()));
    helpLabel.setManaged(StringUtils.isNotEmpty(helpProperty().get()));
    
hence, this will mean making StringUtils.isNotEmpty(helpProperty().get()) return false when there is supposed to be an error message so that on update we do not make helpLabel visible when the error state is not clear yet

Reply:
And for isNotEmpty(helpProperty().get()) to be false, we will have to  set helpLabel.setText("") something like:

validate(): 

  ```
getActiveValidator().ifPresentOrElse(validator -> {
                      errorLabel.setText(validator.getMessage());
                      errorLabel.pseudoClassStateChanged(PSEUDO_CLASS_ERROR, true);
                      helpLabel.setManaged(false);
                      helpLabel.setVisible(false);
                      helpLabel.setText("");
                  },
                  () -> {
                      errorLabel.setText("");
                      errorLabel.pseudoClassStateChanged(PSEUDO_CLASS_ERROR, false);
                      helpLabel.setManaged(true);
                      helpLabel.setVisible(true);
                      helpLabel.setText(globalHelpText);
                  });
```

which means we have to store the help text somewhere first

`String globalHelpText =  "";`

MaterialTextField(@Nullable String description, @Nullable String prompt, @Nullable String help, @Nullable String value): 

```
if (StringUtils.isNotEmpty(help)) {
            helpLabel.setText(help);
            globalHelpText = help;
        }
```

I did this before but figured it was a bit long. If you think this is ok then I will do it